### ISSUE TITLE
Missing Translation on PT

### ISSUE THREAD

**Issue by GabriPalmyro** [wiredashio/wiredash-sdk#298]
I've tried to update my app localization to PT with the CustomWiredashLocation but one property is not overriding.
```
 /// In en, this message translates to:
  /// **'Really? Discard!'**
  String get feedbackDiscardConfirmButton;
  ```
  
  This property is not overriding in PT-Br. How can I change the code to provide the correct translation?

**Comment by passsy** [comment]
You can post the translation here and I'll update it.

You can read about adding it yourself in our [docs](https://docs.wiredash.io/sdk/localization/#contribute-to-wiredash)

**Comment by GabriPalmyro** [comment]
When I'm using `Wiredash.of(Get.context!).showPromoterSurvey(force: true);` using my custom Translation with this code:

```dart
class CustomWiredashTranslationsDelegate
    extends LocalizationsDelegate<WiredashLocalizations> {
  const CustomWiredashTranslationsDelegate();

  @override
  bool isSupported(Locale locale) {
    /// You have to define all languages that should be overridden
    return ['pt'].contains(locale.languageCode);
  }

  @override
  Future<WiredashLocalizations> load(Locale locale) =>
      SynchronousFuture(_PTOverrides());

  @override
  bool shouldReload(CustomWiredashTranslationsDelegate old) => false;
}

class _PTOverrides extends WiredashLocalizationsPt {
  @override
  String get promoterScoreStep1Question =>
      'Depois de utilizar a omni, de 0 a 10, quanto você nos recomendaria para a sua família e amigos?';

  @override
  String get promoterScoreStep1Description =>
      '0 = Não recomendaria, 10 = Recomendaria Muito';
      
  @override
  String get promoterScoreStep2MessageTitle =>
      'O quão provável é você nos recomendar para seus amigos e familiares?';

  @override
  String promoterScoreStep2MessageDescription(int rating) {
    return 'Você poderia nos contar um pouco mais sobre por que você escolheu $rating? Este passo é opcional.';
  }

  @override
  String get promoterScoreStep2MessageHint =>
      'Seria ótimo se você pudesse melhorar...';

  @override
  String get promoterScoreStep3ThanksMessagePromoters =>
      'Obrigado pela sua avaliação!';

  @override
  String get promoterScoreStep3ThanksMessagePassives =>
      'Obrigado pela sua avaliação!';

  @override
  String get promoterScoreStep3ThanksMessageDetractors =>
      'Obrigado pela sua avaliação!';

  @override
  String get promoterScoreNextButton => 'Próximo';

  @override
  String get promoterScoreBackButton => 'Voltar';

  @override
  String get promoterScoreSubmitButton => 'Enviar';

  @override
  String get backdropReturnToApp => 'Voltar ao app';

  @override
  String get feedbackDiscardConfirmButton => 'Você tem certeza?';
}
```

But when my prometerScore appears in the screen the back button after clicking in `close` its showing `Really?` and not `Você tem certeza?` as I override in `feedbackDiscardConfirmButton`.

After looking into the code of this page I found that this translation is not being overrided.

**Comment by passsy** [comment]
Whoops, that's an [oversight](https://github.com/wiredashio/wiredash-sdk/blob/32ffd443032c22b9312da38a24ffc06b5cdc766d/lib/src/core/widgets/backdrop/step_page_scaffold.dart#L237). Thanks for letting us know

**Comment by passsy** [comment]
It is fixed in wiredash 1.7.4

Thanks again for the report
